my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html

```diff
@@ -1,49 +1,65 @@
       </div>
-      <div ng-switch-default>
-        <md-button aria-label="Update phone number" class="md-default md-raised">Update phone number</md-button>
+      <!-- Empty state -->
+      <div ng-if="emergencyPhoneNumbers.emergencyPhoneNumbers.length === 0 && !editMode" >
+        <p><md-icon>warning</md-icon> No phone number entered yet.</p>
+        <md-button aria-label="Add phone number" class="md-accent md-raised no-margin-left" ng-click="edit()">Add phone number</md-button>
       </div>
-    </div>
-  </md-card-content>
-</md-card>
+      <!-- Non-empty -->
+      <div ng-if="!editMode && emergencyPhoneNumbers.emergencyPhoneNumbers.length" layout="row" layout-align="space-between center">
+        <span class="md-subhead" flex="80">
+          {{ emergencyPhoneNumbers.emergencyPhoneNumbers[0].value}} ({{emergencyPhoneNumbers.emergencyPhoneNumbers[0].type }})
+        </span>
+        <md-button class="md-icon-button" aria-label="edit phone number" ng-click="edit()">
+          <md-icon>mode_edit</md-icon>
+          <md-tooltip md-delay="500">Edit</md-tooltip>
+        </md-button>
+      </div>
+      <!-- Phone number form -->
+      <div ng-hide="!editMode">
+        <form novalidate name="emergencyPhoneNumberForm">
+          <br>
+          <md-input-container class="md-block" flex-gt-sm>
+            <label>Phone number*</label>
+            <input type="tel" required aria-required="true" ng-pattern=/^([a-zA-Z0-9\s!-.]{1,30})$/ ng-maxlength="30" ng-model="emergencyPhoneNumbers.emergencyPhoneNumbers[0].value">
+          </md-input-container>
+          <md-input-container class="md-block" flex-gt-sm>
+            <label>Type*</label>
+            <md-select required aria-required="true" ng-model="emergencyPhoneNumbers.emergencyPhoneNumbers[0].type">
+              <md-option value="mobile">Mobile</md-option>
+              <md-option value="home">Home</md-option>
+              <md-option value="work">Work</md-option>
+            </md-select>
+          </md-input-container>
+          <p>*Required</p>
+          <md-button class="md-accent md-raised no-margin-left" aria-label="Save phone number" ng-click="save()" ng-disabled="emergencyPhoneNumberForm.$invalid">Save</md-button>
+          <md-button class="md-default md-raised" aria-label="Cancel" ng-click='cancel()'>Cancel</md-button>
+        </form>
+      </div>
+      <!-- WiscAlert notice -->
+      <div class="notice-container" layout="row" layout-align="start center">
+        <div class="notice-icon" flex-gt-xs="20" hide-xs>
+          <md-icon>info</md-icon>
+        </div>
+        <div class="notice-description" flex-gt-xs="80">
+          <a class="md-subhead" href="https://my.wisc.edu/portal/p/wisc-alerts-text-messaging/render.uP" rel="nofollow noreferrer" target="_blank">WiscAlerts</a>
+          <p>By saving your emergency phone number, you will be automatically signed up for WiscAlerts, UW-Madison's emergency notification system.</p>
```

Comment:
This is okay. Would phrasing it in terms of what happens to the phone number be clearer?

"This phone number will be registered to receive WiscAlerts, UW-Madison's emergency notifications."

Reply:
Yeah, I like your version better.